inc/rlottie.h

```diff
@@ -457,6 +457,19 @@ class RLOTTIE_API Animation {
      */
     const LayerInfoList& layers() const;
 
+    /**
+     *  @brief Returns Layer information{Type, Layer keypath} of fill, stroke object layers of the composition.
+     *         If fill object in the layer1(ind: 2)->group1->fill1 hirarchy,
+     *         it returns {"Fill", "2::layer1::group1::fill1"}
+     * 
+     * 
+     *  @return List of Layer information of the Composition.
+     * 
+     *  @see LayerTypeList
+     *  @internal
+     */
+    const LayerTypeList& allLayersInfoList() const;
```

Comment:
Unfortunately we can't add any public api just for online viewer. So if you need this functionality in the Online viewer . you need to parse the JSON file in the javascript and get all the details.

Reply:
what do you think use c api for getting layer info?